src/pyhepmc/view.py

```diff
@@ -151,3 +151,12 @@ def to_dot(
         )
 
     return d
+
+def save_event(evt: pyhepmc.GenEvent, name: str ="event.png"):
```

Comment:
`name` should be a [os.PathLike](https://docs.python.org/3/library/os.html#os.PathLike) instead of string.

Reply:
I think we should rename this function to `savefig` to allude to matplotlib and to make clear that it only works for saveing as image.

Reply:
Ideally, `name` should also accept a file handle from a file that was already opened.